File: answers/problem1/problem1.c
#include <stdio.h>

void main() {
	char name[20];
	FILE* input = fopen("input.txt", "r");

	while(fscanf(input, "%s", name) != EOF) {
		printf("Hello %s!\n", name);
	}

	fclose(input);
}

File: answers/problem1/Problem1.java
import java.io.*;

public class Problem1 {
	public static void main(String[] args) {
		try {
			BufferedReader br = new BufferedReader(new FileReader(new File("input.txt")));

			for (String line = br.readLine(); line != null; line = br.readLine())
				System.out.println("Hello " + line + "!");

			br.close();
		}
		catch(Exception e) {
			System.out.println(e.getMessage());
		}
	}
}
